frontend/components/ifixit-image/ifixit-image.tsx

```diff
@@ -1,19 +1,20 @@
-import Image, { ImageProps, ImageLoader } from 'next/image';
+import Image, { ImageProps, ImageLoader, ImageLoaderProps } from 'next/image';
 
 export function IfixitImage(props: ImageProps) {
-   let unoptimized = props.unoptimized;
+   const unoptimized = props.unoptimized;
+   let loader;
 
    if (typeof props.src === 'string') {
       if (isGuideImage(props.src)) {
-         unoptimized = true;
+         loader = guideImageLoader;
       } else if (isCartImage(props.src)) {
-         unoptimized = true;
+         loader = cartImageLoader;
       } else if (isStrapiImage(props.src)) {
-         unoptimized = true;
+         // Use default loader unless unoptimized
       }
```

Comment:
I don't think we need this `else if` clause anymore? I think we probably don't need the comment

Reply:
[61dbc35](https://github.com/iFixit/react-commerce/pull/537/commits/61dbc35a2c5a6d8f8818994c8c25225ae159ef12)